packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -333,6 +333,26 @@ export interface IContainerRuntimeBaseExperimental extends IContainerRuntimeBase
 	enterStagingMode?(): StageControlsExperimental;
 	readonly inStagingMode?: boolean;
 }
+/**
+ * These policies can be set by the author of the data store via its data store runtime to influence behaviors.
+ *
+ * @remarks
+ * Policies allow data store authors to define specific behaviors or constraints for their data stores.
+ * These settings can impact how the data store interacts with the runtime and other components.
+ *
+ * @legacy
+ * @alpha
+ */
+export interface IFluidDataStorePolicies {
+	/**
+	 * When set to true, data stores will be set to readonly while in staging mode.
+	 *
+	 * @remarks
+	 * This policy is useful for data stores that do not support staging mode, such as those using consensus DDS.
+	 * It ensures that the data store remains readonly during staging mode to prevent unsupported operations.
+	 */
+	readonly readonlyInStagingMode: boolean;
```

Comment:
Where did we land with "strict mode"? Do we want to give this 3 values: "NO" | "YES" | "STRICT" (naming whatever) and actually block changes when set to STRICT?

Reply:
i hadn't actually thought about bundling strict mode into this change. i think i might do that. good idea.

Reply:
turns out this is pretty hard right now, as we don't have a way to signal the container to close.